src/Modules/BrainStimulator/SetupRHSforTDCSandTMS.cc

```diff
@@ -63,6 +63,8 @@ void SetupTDCS::setStateDefaults()
   setStateDoubleFromAlgo(Parameters::normal_dot_product_bound);
   setStateDoubleFromAlgo(Parameters::pointdistancebound);
   setStateIntFromAlgo(Parameters::number_of_electrodes);
+  setStateBoolFromAlgo(Parameters::GetContactSurface);
+  state->setValue(Parameters::GetContactSurface, false);
```

Comment:
@moritzdannhauer why this line? you just overwrote the value from the network file.